KIF Tests/TableViewTests.m

```diff
@@ -83,6 +83,20 @@ - (void)testTappingRowUnderToolbarByLabel
     [tester tapViewWithAccessibilityLabel:@"Cell 32"];
 }
 
+- (void)testWaitingRowByLabel
```

Comment:
Please add the equivalent tests for `TableViewTests_ViewTestActor.m`. We'd like to shift people towards using the new syntax and should have test parity between `viewTester` and `tester`.